Source/UI/NewUI/PaymentForm/Model/PaymentFormConfiguration.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  PaymentFormConfiguration.swift
+//  
+//
+//  Created by Alex Ioja-Yang on 26/07/2022.
+//
+
+import Foundation
+
+public struct PaymentFormConfiguration {
```

Comment:
👍

Reply:
If any changes made to public configuration should be agreed with devs and also updated on the API spec.

Reply:
refer : https://checkout.atlassian.net/wiki/spaces/FS/pages/5266440242/Draft+Frames+Component+API+Spec

Reply:
Please advise me on what to do and what requirements are there to be met ❓

Reply:
Looking at that doc its so out of sync with the state of frames that I might not be the best person to add this change there. There's just no reference point to associate this with 😵

Reply:
@ehab-al-cko can help.